Tests/test_file_mic.py

```diff
@@ -1,10 +1,46 @@
-from helper import unittest, PillowTestCase
+from helper import unittest, PillowTestCase, hopper
 
-from PIL import MicImagePlugin
+from PIL import Image, MicImagePlugin
+
+TEST_FILE = "Tests/images/hopper.mic"
 
 
 class TestFileMic(PillowTestCase):
 
+    def test_sanity(self):
+        im = Image.open(TEST_FILE)
+        im.load()
+        self.assertEqual(im.mode, "RGBA")
+        self.assertEqual(im.size, (128, 128))
+        self.assertEqual(im.format, "MIC")
+
+        im2 = hopper("RGBA")
+        self.assert_image_similar(im, im2, 123.5)
```

Comment:
Ok, there's a gamma of 2.2 encoded into the file, as per some of the defaults in the Tiff spec, but there's no indication in the tags that there is a gamma applied. With these changes, we get a much tighter epsilon=10:

```
        lut = ImagePalette.make_gamma_lut(1/2.2)
        im = Image.merge('RGBA', [chan.point(lut) for chan in im.split()])

        im2 = hopper("RGBA")
        self.assert_image_similar(im, im2, 10)
```

This is perhaps not the place to do it, but it does explain the results. Also, I think we need some form of 'apply gamma' function on an image, because splitting and recombining should be something we encapsulate, not require.